src/bridge/SequencerInbox.sol

```diff
@@ -455,9 +470,18 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         uint256 afterDelayedMessagesRead,
         IGasRefunder gasRefunder,
         uint256 prevMessageCount,
-        uint256 newMessageCount
+        uint256 newMessageCount,
+        bytes memory quote
     ) external override refundsGas(gasRefunder, IReader4844(address(0))) {
         if (!isBatchPoster[msg.sender] && msg.sender != address(rollup)) revert NotBatchPoster();
+        //  If address is rollup skip the batch poster check
+        if (isBatchPoster[msg.sender]) {
```

Comment:
only checking this if the batch has been posted by the batch poster because rollup admin is not running in a TEE and thus, cannot produce a quote.

Reply:
Inline this comment? The current comment only describes what the code does instead of why.